Tests/powershellcore/U033.PSCoreScripts.Invoke-SmokeTestOnWebApp.Tests.ps1

```diff
@@ -0,0 +1,185 @@
+﻿Push-Location -Path $PSScriptRoot\..\..\PSCoreScripts\
+
+Describe "Invoke-SmokeTests unit tests" -Tag "Unit" {
+
+    function Invoke-WebRequest { param($Method, $Uri, $MaximumRedirection, $TimeoutSec, [switch] $UseBasicParsing) }
```

Comment:
Do we need this. I thought Invoke-WebRequest (like Start-Sleep) was part of the core not loaded from a module

Reply:
We didn't - and the Az module allows us to properly mock Get-AzWebAppSlot, so I've also removed the function definition for that too.